internal/ini/ini.go

```diff
@@ -34,13 +34,13 @@ func Parse(f io.Reader) (Sections, error) {
 }
 
 // ParseBytes will parse the given bytes and return the parsed sections.
-func ParseBytes(b []byte) (Sections, error) {
+func ParseBytes(b []byte, path string) (Sections, error) {
 	tree, err := ParseASTBytes(b)
 	if err != nil {
 		return Sections{}, err
 	}
 
-	v := NewDefaultVisitor()
+	v := NewDefaultVisitor(path)
```

Comment:
suggest changing `path` here to empty string and not add extra parameter to `ParseBytes`